### PR TITLE
Read sensors in config

### PR DESCRIPTION
**Pull request by jakeshepherd**

### CODE REVIEW

File: tests/Unit/SensuConfigServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        mkDir(__DIR__ . '/sensor-config');
+        mkDir(__DIR__ . '/sensor-config/iaptus-uk-test');
+        mkDir(__DIR__ . '/sensor-config/iaptus-aus-test');
+
+        $file = fopen(__DIR__ . '/sensor-config/iaptus-uk-test/test-sensor.json', 'w+');
+        fwrite($file, json_encode(['sensor-uk' => 'green']));
+        fclose($file);
+
+        $file = fopen(__DIR__ . '/sensor-config/iaptus-aus-test/test-sensor.json', 'w+');
+        fwrite($file, json_encode(['sensor-ca' => 'red']));
```
Comment (Geoff10) [inline tests/Unit/SensuConfigServiceTest.php:23]: Is this supposed to be `ca` not `aus`?

**Review by Geoff10 — CHANGES_REQUESTED** [review]
Just one question really.

File: tests/Unit/SensuConfigServiceTest.php (hunk 1 above)
Comment (jakeshepherd) [reply tests/Unit/SensuConfigServiceTest.php:23]: Yes :( a86d62f2d495719321ce5ae43dbb9c53be4f5e95

**Review by Geoff10 — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+language: php
+
+php:
+  - 7.3
```
Comment (scott-evans-mayden) [inline .travis.yml:4]: Just a question - have you tried any newer versions?
Given 7.3 is no longer in support, 7.4 is just ending now too! 😭

Comment (jakeshepherd) [reply .travis.yml]: just tried this https://app.travis-ci.com/github/may-den/sensu-dashboard/jobs/587720326
8.0 and 8.1 fail but 7.4 works :) [1ce6671](https://github.com/may-den/sensu-dashboard/pull/30/commits/1ce667124baa4ff5f8e4fdbc9420b94aa2ed922e)

**Review by Geoff10 — APPROVED** [review]

**Review by scott-evans-mayden — APPROVED** [review]

**Review by scott-evans-mayden — APPROVED** [review]

**Review by Geoff10 — APPROVED** [review]

**Pull request merged**